src/app/cli/src/cli_commands.rs

```diff
@@ -546,3 +546,19 @@ where
 
     Ok(result_refs)
 }
+
+// ToDo add validation for DatasetRefPattern and DatasetRef expresion
+fn validate_many_dataset_wildcards<I>(
+    _catalog: &dill::Catalog,
+    dataset_refs: I,
+) -> Result<Vec<String>, CLIError>
+where
+    I: IntoIterator<Item = String>,
+{
+    let mut result_refs = Vec::new();
+    for dataset_ref in dataset_refs.into_iter() {
+        result_refs.push(dataset_ref);
+    }
+
+    return Ok(result_refs);
+}
```

Comment:
What exactly this would validate?
In non-wildcard version we are checking for dataset existence in the workspace.
In wildcard this is not applicable.

I assume there might be nothing to validate. We just need to ensure parsing of arguments in value parser is done correctly.